manifests/bootstrap-manifests/bootstrap-apiserver.yaml

```diff
@@ -0,0 +1,48 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: bootstrap-kube-apiserver
+  namespace: {{ .BootstrapKubeAPIServerConfig.Namespace }}
+  labels:
+    openshift.io/control-plane: true
+    openshift.io/component: api
+  annotations:
+    openshift.io/run-level: 0
+spec:
+  restartPolicy: Always
+  hostNetwork: true
+  containers:
+  - name: kube-apiserver
+    image: {{ .BootstrapKubeAPIServerConfig.Image }}
+    imagePullPolicy: {{ .BootstrapKubeAPIServerConfig.ImagePullPolicy }}
+    command: ["/bin/bash", "-c"]
+    args:
+    - |
+      #!/bin/bash
+      set -euo pipefail
+      if [[ -f /etc/origin/master/master.env ]]; then
+        set -o allexport
+        source /etc/origin/master/master.env
+      fi
+      exec hypershift openshift-kube-apiserver --config=/etc/origin/master/config.json
+    securityContext:
+      privileged: true
+    volumeMounts:
+    - mountPath: /etc/origin/master/
+      name: master-config
+    - mountPath: /etc/origin/cloudprovider/
+      name: master-cloud-provider
+    - mountPath: /var/lib/origin/
+      name: master-data
+    livenessProbe:
+      httpGet:
+        scheme: HTTPS
+        port: 8443
+        path: healthz
+  volumes:
+  - name: master-config
+    hostPath:
+      path: {{ .ConfigDir }}
```

Comment:
now I understand. Why is this configurable? IMO it shouldn't be. No need to complicate the render command. Let's just fix this and output the right manifest.

In other words: the flags of the render command should only tweak input + output dirs of the render command, not influence the manifests.

Reply:
You might want a way to tweak this in cluster up, right? ;-)

Reply:
hmm, yes. Always cluster-up that makes everything ugly ;-)